docs/topics/auth/default.txt

```diff
@@ -500,6 +500,41 @@ The login_required decorator
     :func:`django.contrib.admin.views.decorators.staff_member_required`
     decorator a useful alternative to ``login_required()``.
 
+.. currentmodule:: django.contrib.auth.mixins
+
+The ``LoginRequired`` mixin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using :doc:`class-based views </topics/class-based-views/index>`, you can
+achieve the same behavior as with ``login_required`` by using the
+``LoginRequiredMixin``. This mixin should be at the leftmost position in the
+inheritance list.
+
+.. class:: LoginRequiredMixin
+
+    .. versionadded:: 1.9
+
+    If a view is using this mixin, all requests by non-authenticated users
+    will be redirected to the login page or are shown a HTTP 403 Forbidden
```

Comment:
By the way, it is correct to say "an HTTP", because in this case we have an unsounded "h", even though "h" is a consonant. But it's a minor thing, you can leave it as is :)